src/vm/jitinterface.cpp

```diff
@@ -1302,9 +1302,23 @@ struct TryResolveTokenFilterParam
 
 bool isValidTokenForTryResolveToken(CEEInfo* info, CORINFO_RESOLVED_TOKEN* resolvedToken)
 {
-    LIMITED_METHOD_CONTRACT;
+    CONTRACTL {
+        NOTHROW;
+        GC_TRIGGERS;
```

Comment:
(FWIW, `MDInternalRO::IsValidToken` should not throw, trigger a GC, etc. from inspection)